src/main/java/core/basesyntax/FileWork.java

```
@@ -9,6 +15,26 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        if (fileName.isEmpty()) {
+            return new String[]{};
+        }
+        FileReader fr = null;
+        try {
+            fr = new FileReader(fileName);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
```

don't ignore exceptions